core/stream.h

```diff
@@ -1,49 +1,25 @@
 #pragma once
 
+#include "json.h"
+#include "memory.h"
+#include "view.h"
 #include "dataset.h"
-#include <algorithm>
-#include <limits>
 #include <vector>
 
-struct counter {
-    using value_type = std::uint8_t;
-    using pointer = typename std::add_pointer<value_type>::type;
-    using const_pointer = typename std::add_pointer<const value_type>::type;
-    using iterator = std::vector<value_type>::iterator;
-    using const_iterator = std::vector<value_type>::const_iterator;
+using value_type = std::uint8_t;
```

Comment:
I know, but I am lazy to fix it. I solve things like you do, someone else can solve my code.